src/CommunityToolkit/Xamarin.CommunityToolkit/AttachedProperties/SetFocusOnEntryCompleted.shared.cs

```diff
@@ -36,12 +37,17 @@ public static void SetNextElement(BindableObject view, VisualElement value)
 
 		static void OnNextElementChanged(BindableObject bindable, object oldValue, object newValue)
 		{
-            var entry = bindable as Entry;
+			var entry = (Entry)bindable;
+			var weakEntry = new WeakReference<Entry>((Entry)bindable);
+			entry.Completed += completedHandler;
```

Comment:
I am not familiar with WeakReference but as @pictos mentioned I believe you should use [WeakSubscribe()](https://github.com/xamarin/XamarinCommunityToolkit/blob/0d55a6340508de0bd63be4c44d484e0823529d5c/src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/INotifyPropertyChangedEx.shared.cs#L8) somehow because we are still subscribing to the event in this line and not unsubscribing from it, right @pictos ?